CMakeLists.txt

```diff
@@ -5,8 +5,8 @@ include(CheckCCompilerFlag)
 include(GenerateExportHeader)
 include(GNUInstallDirs)
 
-option(BUILD_TESTS "Build test programs" ON)
-option(BUILD_SHARED_LIBS "Build shared library" ON)
+option(BUILD_TESTS "Build test programs" OFF)
+option(BUILD_SHARED_LIBS "Build shared library" OFF)
```

Comment:
I prefer to have these changes.

Reply:
Ah, yes, this was just a mistake on my part to add a local cmake file in. I will remove that